### PR TITLE
Add daily digest

### PR DESCRIPTION
**Pull request by jawwadmukhtargmail**

### CODE REVIEW

File: RequestService/RequestService.Core/Services/DailyDigestService.cs [hunk 1]
```diff
@@ -0,0 +1,145 @@
...
+
+            users.UserDetails = users.UserDetails.Where(x => x.SupportRadiusMiles.HasValue);
+
+            foreach (var user in users.UserDetails)
+            {
+                try
+                {
+                    var activitySpecificSupportDistancesInMiles = nationalSupportActivities.Where(a => user.SupportActivities.Contains(a)).ToDictionary(a => a, a => (double?)null);
+
+                    var attachedDistances = await _jobFilteringService.FilterJobSummaries(openRequests, null, user.PostCode, _applicationConfig.Value.DistanceInMilesForDailyDigest, activitySpecificSupportDistancesInMiles, null, null, new List<JobStatuses>() { JobStatuses.Open }, cancellationToken);
```
Comment (mike-f50) [inline RequestService/RequestService.Core/Services/DailyDigestService.cs:72]: Do we also need to pass in groups to filter on?

File: RequestService/RequestService.Core/Services/DailyDigestService.cs [hunk 2]
```diff
@@ -0,0 +1,145 @@
...
+
+                    //if they dont have the community connector support activity, let remove any open requests in there.
+                    if (!user.SupportActivities.Contains(SupportActivities.CommunityConnector))
```
Comment (mike-f50) [inline RequestService/RequestService.Core/Services/DailyDigestService.cs:75]: This if block can go (providing we are filtering on group).

Comment (jawwadmukhtargmail) [reply RequestService/RequestService.Core/Services/DailyDigestService.cs]: done

File: RequestService/RequestService.Core/Services/DailyDigestService.cs (hunk 1 above)
Comment (jawwadmukhtargmail) [reply RequestService/RequestService.Core/Services/DailyDigestService.cs]: done

**Pull request merged**